enclave_manager/avalon_enclave_manager/work_order_processor_delegate.py

```diff
@@ -0,0 +1,225 @@
+
+    def process_work_orders(self):
+        """
+        Executes Run time flow of enclave manager
+        """
+        logger.info("Processing work orders")
+        try:
+            # Get all workorders requests from KV storage lookup and process
+            list_of_workorders = self._kv_helper.lookup("wo-scheduled")
+            if not list_of_workorders:
+                logger.info("Received empty list of work orders from " +
+                            "wo-scheduled table")
+                return
+        except Exception as e:
+            logger.error("Problem while getting keys from wo-scheduled table")
+            return
+
+        for wo_id in list_of_workorders:
+            self.process_work_order_by_id(wo_id)
+        # end of for loop
+
+# -------------------------------------------------------------------------
+
+    def process_work_order_by_id(self, wo_id):
+        """
+        Process the work-order of the specified work-order id. Execute the
+        work-order and update the tables accordingly.
+        Parameters :
+            wo_id - It is the work-order id of the work-order that is to be
+                    processed
+        Returns :
+            wo_resp - A JSON response of the executed work order
+        """
+        try:
+            self._kv_helper.set("wo-processing", wo_id,
+                                WorkOrderStatus.PROCESSING.name)
+
+            # Get JSON workorder request corresponding to wo_id
+            wo_json_req = self._kv_helper.get("wo-requests", wo_id)
+            if wo_json_req is None:
+                logger.error("Received empty work order corresponding " +
+                             "to id %s from wo-requests table", wo_id)
+                self._kv_helper.remove("wo-processing", wo_id)
+                return None
+
+        except Exception as e:
+            logger.error("Problem while reading the work order %s"
+                         "from wo-requests table", wo_id)
+            self._kv_helper.remove("wo-processing", wo_id)
+            return None
+
+        logger.info("Create workorder entry %s in wo-processing table",
+                    wo_id)
+        self._kv_helper.set("wo-processing", wo_id,
+                            WorkOrderStatus.PROCESSING.name)
+
+        logger.info("Delete workorder entry %s from wo-scheduled table",
+                    wo_id)
+        self._kv_helper.remove("wo-scheduled", wo_id)
+
+        logger.info("Validating JSON workorder request %s", wo_id)
+        if not self._validate_request(wo_id, wo_json_req):
+            return None
+
+        # Execute work order request
+
+        logger.info("Execute workorder with id %s", wo_id)
+        wo_json_resp = self._wo_execute_callback(wo_json_req)
+        wo_resp = json.loads(wo_json_resp)
+
+        logger.info("Update workorder receipt for workorder %s", wo_id)
+        self._wo_kv_delegate.update_receipt(wo_id, wo_resp)
+
+        if "Response" in wo_resp and \
```

Comment:
Do we have field named "Response" in the work order response in case of failures?